### ISSUE TITLE
Simplify workflow for setting affiliations

### ISSUE THREAD

**Issue by bemoody** [MIT-LCP/physionet-build#978]
Two small things that might be nice:

- When someone receives an invitation to be a coauthor, require them to supply their affiliation as part of the form to accept the invitation.

- Include a check box on that form to "save as my default affiliation".

Make it a little bit simpler for those non-submitting authors who don't really know their way around the project submission process, and may not have any interest in learning.

**Comment by tompollard** [comment]
> When someone receives an invitation to be a coauthor, require them to supply their affiliation as part of the form to accept the invitation.

Agree, this would be a big improvement! Too many people miss the affiliation step currently.
